models/repo.go

```diff
@@ -275,12 +275,35 @@ func (repo *Repository) innerAPIFormat(e Engine, mode AccessMode, isParent bool)
 		}
 	}
 	hasIssues := false
-	if _, err := repo.getUnit(e, UnitTypeIssues); err == nil {
+	var externalTracker *api.ExternalTracker
+	var internalTracker *api.InternalTracker
+	if unit, err := repo.getUnit(e, UnitTypeIssues); err == nil {
+		config := unit.IssuesConfig()
 		hasIssues = true
+		internalTracker = &api.InternalTracker{
+			EnableTimeTracker:            config.EnableTimetracker,
+			LetOnlyContributorsTrackTime: config.AllowOnlyContributorsToTrackTime,
```

Comment:
How about use `AllowOnlyContributorsToTrackTime` but not `LetOnlyContributorsTrackTime`. I found only this variable is not the same as config.

Reply:
I changed this. 

There is also a slight discrepancy in EnableIssueDependencies - EnableDependencies but the former is more descriptive so maybe there is a point keeping it.